src/main/java/tracko/model/tag/Tag.java

```diff
@@ -9,7 +9,8 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric and "
+            + "should not be more than 30 characters long.";
```

Comment:
Little grammatical error here --> `Tags name` should be `A tag's name` or `Tags' names`. Think the former would be good!

Reply:
Thanks for pointing it out!